### DISCUSSION TITLE
Setting color in tables

### DISCUSSION THREAD

**Discussion by lozdown** [sqlpage/SQLPage#552] [Q&A] [answered]
I need some help to work out what color specifications can be used for the ` _sqlpage_color` table row attribute. 
As examples:

`select .... 'red' as _sqlpage_color` correctly sets rows red, 

however `select .... 'darkgray' as _sqlpage_color` or  `select .... '#DCDCDC' as _sqlpage_color` have no effect.

**Comment by lovasoa** [comment, marked as answer]
Hello! the list of colors is here: https://sql.datapage.app/colors.sql

I updated the documentation
